lookout/style/format/features.py

```diff
@@ -232,9 +243,12 @@ def extract_features(self, files: Iterable[File], lines: List[List[int]]=None
             try:
                 vnodes, parents = self._parse_file(contents, uast, file.path)
             except AssertionError as e:
-                self._log.warning("could not parse file %s with error \"%s\', skipping",
+                self._log.warning("could not parse file %s with error '%s', skipping",
                                   file.path, e)
-                continue
+                if self.debug_parsing:
```

Comment:
Alternative is to use https://github.com/src-d/ml/blob/69361199e04f393d1a10b10badb969a3f5b2ed24/sourced/ml/utils/engine.py#L83-L92

Reply:
true but I feel like it's more verbose, needs an import and doesn't filter on exception type so maybe it's better to stick to that